File: appmy/src/main/java/com/leilei/guoshujinfu/mylearning/util/AppToast.java
package com.leilei.guoshujinfu.mylearning.util;

import android.content.Context;
import android.view.Gravity;
import android.widget.Toast;

/**
 * Created by Administrator on 2017/8/3.
 */

public final class AppToast extends Toast {
    public AppToast(Context context) {
        super(context);
    }

    public static void toastShow(Context context, String content) {
        Toast toast = Toast.makeText(context, content, Toast.LENGTH_SHORT);
        toast.setGravity(Gravity.CENTER, 0, 0);
        toast.show();
    }

    public static void toastShow(Context context, String content, int duration) {
        Toast toast = Toast.makeText(context, content, duration);
        toast.setGravity(Gravity.CENTER, 0, 0);
        toast.show();
    }

}

File: aqrleilib/src/main/kotlin/com/aqrairsigns/aqrleilib/util/AppToast.kt
package com.aqrairsigns.aqrleilib.util

import android.content.Context
import android.view.Gravity
import android.widget.Toast

/**
 *@Author: AqrLei
 *@Date: 2017/8/22
 */
object AppToast {
    fun toastShow(context: Context, content: String, time: Int=1000, gravity: Int = Gravity.CENTER) {
        val toast = Toast.makeText(context, content, time)

        toast.setGravity(gravity, 0, 0)
        toast.show()
    }
}